packages/automerge-repo-svelte-store/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@automerge/automerge-repo-svelte-store",
   "version": "2.0.0-alpha.20",
-  "description": "A Svelte store containing your automerge documentsj",
+  "description": "A Svelte store containing your automerge documents",
   "repository": "https://github.com/automerge/automerge-repo/tree/master/packages/automerge-repo-svelte-store",
   "license": "MIT",
   "type": "module",
   "main": "dist/index.js",
   "scripts": {
-    "build": "tsc",
+    "build": "",
```

Comment:
right, i forgot i need some svelte help

Reply:
i think it's fine to ship an alpha but i need to either get help or learn some svelte i suppose